packages/widgetbook/lib/src/fields/string_field.dart

```diff
@@ -14,10 +14,11 @@ class StringField extends Field<String> {
   }) : super(
           type: FieldType.string,
           codec: FieldCodec(
-            toParam: (value) => Uri.encodeComponent(value),
-            toValue: (param) => param != null
-                ? Uri.decodeComponent(param) //
-                : null,
+            toParam: (value) => value,
```

Comment:
**issue:** Since `Uri.encodeComponent` is removed from here, this means that reserved characters (e.g. commas) will not be encoded. The encoding should be moved to `FieldCodec.encodeQueryGroup` to keep the encoding as expected.

Reply:
In your PR also removed toValue -> decoding, toParam -> encoding, because encoding, decoding performed in `encodeQueryGroup` and `decodeQueryGroup`

Reply:
It was resolved in the other PR 👍